Services/Component/classes/Setup/class.ilComponentDefinitionsStoredObjective.php

```diff
@@ -61,6 +63,12 @@ public function achieve(Setup\Environment $environment) : Setup\Environment
         $db = $environment->getResource(Setup\Environment::RESOURCE_DATABASE);
         $ini = $environment->getResource(Setup\Environment::RESOURCE_ILIAS_INI);
         $client_ini = $environment->getResource(Setup\Environment::RESOURCE_CLIENT_INI);
+        /** @var ilComponentRepository $component_repository  */
+        $component_repository = $environment->getResource(Setup\Environment::RESOURCE_COMPONENT_DATABASE);
```

Comment:
I [changed the name to `RESOURCE_COMPONENT_REPOSITORY`](https://github.com/ILIAS-eLearning/ILIAS/commit/c88e42946608716a7fc6bc92cabcc2330a1435a9) to implement our [repository naming policy](https://github.com/ILIAS-eLearning/ILIAS/blob/trunk/docs/development/repository-pattern.md#conventions).

Reply:
Thanks a lot for your review @klees . I'll tackle your remarks,